cameras:
    resolution_arguments:
        "144":
            "--headless"
            "--bitrate=300000"
            "--width=256"
            "--height=144"
        "360":
            "--headless"
            "--bitrate=800000"
            "--width=480"
            "--height=360"
        "720":
            "--headless"
            "--bitrate=1800000"
            "--width=1280"
            "--height=720"
    default_mission: "erd"
    max_video_device_id_number: 10
    missions:
        auton:
            ips:
                "10.0.0.1:5000"
                "10.0.0.1:5001"
            resolution: "144"
        erd:
            ips:
                "10.0.0.1:5000"
                "10.0.0.1:5001"
            resolution: "144"
        es:
            ips:
                "10.0.0.1:5000"
                "10.0.0.1:5001"
            resolution: "720"
        science:
            ips:
                "10.0.0.1:5000"
                "10.0.0.1:5001"
                "10.0.0.2:5000"
                "10.0.0.2:5001"
            resolution: "144"
    # If number_of_pipelines is changed,
    # then ChangeCameras.srv must be changed too
    number_of_pipelines: 4
